### DISCUSSION TITLE
Compilation error on the default 'car' sample application

### DISCUSSION THREAD

**Discussion by rsys-spalanis** [freeconf/restconf#43] [General]
Hi,

I tried running the sample car application and followed the instruction as defined in the github instructions. But it fails with below error.

car# go run .
# github.com/freeconf/restconf/device
/root/go/pkg/mod/github.com/freeconf/restconf@v0.0.0-20230602153529-4e888964741d/device/local.go:101:78: b.Root().UpsertFromSetDefaults(nodeutil.ReflectChild(moduleCfg)).LastErr undefined (type error has no field or method LastErr)
/root/go/pkg/mod/github.com/freeconf/restconf@v0.0.0-20230602153529-4e888964741d/device/yang_lib.go:17:12: multiple-value ietfYangLib.Root().Find("modules-state/module") (value of type (*node.Selection, error)) in single-value context

Please let me know if I'm missing something.

Regards,
Suresh

**Comment by dhubler** [comment]
This was a problem! Thank for reporting.  Should be fixed now, remove "car" directory and start over.  

Issue: restconf repo was pointing to the wrong version of yang repo.

**Reply by rsys-spalanis** [reply]
Thank you so much for quickly looking into the issue. It is working now.

Truly appreciate your help.

Regards
Suresh
